docs/app/pages/css/css-sections/icons/ux-icons/hpe-icons-link.txt

```diff
@@ -0,0 +1 @@
+<link rel="stylesheet" href="@ux-aspects/ux-aspects/dist/styles/hpe-icons.min.css">
```

Comment:
Almost everyone will be using the Angular CLI, in which case this won't work. We should recommend adding it to the `angular.json` file under the `styles` property. An alternative can also be to use the `@import` within their global stylesheet.